aptos-move/aptos-release-builder/data/freeze_fix.yaml

```diff
@@ -0,0 +1,13 @@
+---
+remote_endpoint: ~
+name: "v1.16-freeze_code_object-fix"
+proposals:
+  - name: step_1_upgrade_framework
+    metadata:
+      title: "v1.16 code::freeze_code_object fix"
```

Comment:
Is this going to replace the main title for the v1.16 framework upgrade? If so this should be removed

Reply:
We have `release.yaml` for the framework upgrade on the mainnet. 
`freeze_fix.yaml` is just for the testnet framework upgrade as we need to patch the testnet framework for the freeze fix.